packages/IMAPClient-Core.package/ICFileReadWriter.class/README.md

```diff
@@ -0,0 +1 @@
+An ICFileReadWriter is a JSON File Writer and Reader to implement the file system storage of mails
```

Comment:
I would rather describe this as *wrapper for file system storage of mails*.

Reply:
I do prefer my wording as I think it describes the class better.
@Paulpanther @martintaraz @felixrindt

Reply:
I think Tims version is better, since it has more info

Reply:
Okay, but please change it to *... Writer and Reader implementing the file system storage of mails.*.

Reply:
Okay, I kept @EatingBacon's version but changed the term *implement* to *handle* due to this describing what an instance of ICFileReadWriter does.